manager/assets/modext/widgets/security/modx.panel.user.group.js

```diff
@@ -105,7 +105,7 @@ MODx.panel.UserGroup = function(config) {
                                 ,fieldLabel: _('user_group_parent')
                                 ,editable: false
                                 ,anchor: '100%'
-                                ,disabled: config.record.id === 0
+                                ,disabled: config.record.id === 0 || config.record.name === 'Administrator'
```

Comment:
Is it "Administrator" in all languages?

Reply:
Same concept is used on line 69 for disabling changing name.

Reply:
Shouldn't we use a lexicon instead? Since you're using it in the `isAdminGroup()` function as well:

https://github.com/modxcms/revolution/blob/92931b6d622a9860db63c23dc6794ef755c3c939/core/src/Revolution/Processors/Security/Group/Update.php#L191

Reply:
Fair, using lexicons makes no sense. Adjusted.